RileyLinkKitTests/PumpOpsSynchronousBuildFromFramesTests.swift

```diff
@@ -74,16 +74,14 @@ class PumpOpsSynchronousBuildFromFramesTests: XCTestCase {
         assertThrows(try _ = sut.getHistoryEvents(since: Date()))
     }
     
-    func testAllEventsReturned() {
+    func test332EventsReturnedUntilOutOrder() {
```

Comment:
Yes, but then you changed the time, and that's a change time event that we'd see.  I should probably look at the history page in the test and see what's going on.